[LGThinq] Enforce capitalized `manualCountry`

With openHAB 5.1.4 I had hard times to get the LG ThinQ Gateway  (Bridge) online.  It turned out, that I had to type the `manualCountry` property in capitals.

• Enforce that the property `manualCountry` is submitted to the LG server in capitals

Otherwise the log below happens, which is impossible to understand.
```
2026-06-16 18:32:00.975 [TRACE] [ing.lgthinq.lgservices.api.RestUtils] - POST request to URI: https://us.emp.lgsmartplatform.com/emp/v2.0/account/session/abc%40domain.be
2026-06-16 18:32:01.448 [TRACE] [ing.lgthinq.lgservices.api.RestUtils] - POST response: {                       
  "error" : {                                                                                                   
    "request" : "/emp/v2.0/account/session/abc%40domain.be",                                      
    "code" : "MS.001.03",                                                                                       
    "message" : "BG"                                                                                            
  }                                                                                                             
}                                                                                                               
2026-06-16 18:32:01.450 [DEBUG] [nternal.handler.LGThinQBridgeHandler] - Error accessing LG API. Updating Bridge
 Status to OFFLINE.                                                                                             
org.openhab.binding.lgthinq.lgservices.errors.AccountLoginException: Error doing user's account login on the Emp
 LG Server                                                                                                      
        at org.openhab.binding.lgthinq.lgservices.api.TokenManager.oauthFirstRegistration(TokenManager.java:111)
 ~[?:?]                                                                                                         
        at org.openhab.binding.lgthinq.internal.handler.LGThinQBridgeHandler$PollingRunnable.run(LGThinQBridgeHa
ndler.java:276) ~[?:?]                                                                                          
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572) ~[?:?]                       
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358) ~[?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.
java:305) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
        at java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.IllegalStateException: Error login into account. The reason is: {
  "error" : {
    "request" : "/emp/v2.0/account/session/abc%40domain.be",
    "code" : "MS.001.03",
    "message" : "BG"
  }
}
        at org.openhab.binding.lgthinq.lgservices.api.LGThinqOauthEmpAuthenticator.loginUser(LGThinqOauthEmpAuth
enticator.java:169) ~[?:?]
        at org.openhab.binding.lgthinq.lgservices.api.TokenManager.oauthFirstRegistration(TokenManager.java:109)
 ~[?:?]
        ... 7 more
```

--- a/bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/LGThinQBridgeConfiguration.java
+++ b/bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/LGThinQBridgeConfiguration.java
@@ -12,6 +12,8 @@
  */
 package org.openhab.binding.lgthinq.internal;
 
+import java.util.Locale;
+
 import org.eclipse.jdt.annotation.NonNullByDefault;
 
 /**
@@ -56,7 +58,7 @@ public class LGThinQBridgeConfiguration {
 
     public String getCountry() {
         if ("--".equals(country)) {
-            return manualCountry;
+            return manualCountry.toUpperCase(Locale.ROOT);
         }
         return country;
     }

